iosApp/iosApp/Explore/RepoListView.swift

```diff
@@ -7,23 +7,68 @@
 //
 
 import SwiftUI
+import ExploreShared
 
 struct RepoListView: View {
     @StateObject var viewModel = RepoListViewModel()
 
     var body: some View {
-        ForEach(self.viewModel.repoList, id: \.name) { repo in
-            VStack {
-                Text(repo.name)
-            }.padding(.bottom, 10)
-        }.onAppear {
-            Task {
-                await self.viewModel.loadRepos()
+        let list = self.viewModel.repoList
+        let isLastPage = self.viewModel.isLastPage
+
+        ZStack {
+            ScrollView(.vertical) {
+                LazyVStack {
+                    if !list.isEmpty {
+                        ListView(list)
+                    }
+                    
+                    if !isLastPage {
+                        LoadingView()
+                            .onAppear {
+                                Task {
+                                    await self.viewModel.loadRepos()
+                                }
+                            }
+                    }
+                }
             }
         }
     }
 }
 
+private struct ListView: View {
```

Comment:
It's not common to extract a private subview like this in SwiftUI. We only extract subviews as structs if we want to reuse them in other Views. (They also need to have their own file). If we want to extract the ForEach loop as a small subview to make the code cleaner, exclusive to this RepoListView, it is common practice to declare a computed variable outside the `var body` that returns `some View`. Example: `var listView: some View {
        ForEach(self.viewModel.repoList, id:\.self) { repo in
            RepoView(repo)
        }
    }`